app/views/overview/_list-row-expanded.html

```diff
@@ -2,82 +2,96 @@
   <div class="list-pf-content">
     <alerts alerts="row.notifications"></alerts>
     <div ng-if="row.current">
-      <div class="row">
-        <div ng-if="row.state.breakpoint !== 'xs'">
-          <div ng-class="{
-                'col-sm-7 col-md-8 col-lg-9': !row.state.showMetrics && !row.previous,
-                'col-sm-12 col-md-4 col-lg-5': row.state.showMetrics && !row.previous,
-                'hidden-sm col-md-5 col-lg-5': row.previous
-              }">
-            <!-- TODO: markup semantics suggest div.component-label in <pod-template> should be an h4,
-                       but only here on the overview -->
-            <pod-template
-              pod-template="row.current | podTemplate"
-              images-by-docker-reference="row.state.imagesByDockerReference"
-              builds="row.state.builds"></pod-template>
+      <div class="row-expanded-top" ng-class="{
+            'metrics-active': row.state.showMetrics,
+            'metrics-not-active': !row.state.showMetrics
+      }">
+        <div ng-if="row.state.breakpoint !== 'xs'" class="overview-pod-template" ng-class="{
+          'ng-enter': row.previous,
+          'hidden-sm hidden-md': row.previous && row.state.breakpoint >= 'md'
```

Comment:
I changed this to be `!== 'lg'`, but the other way looked to be working for me. :)